src/views/AlbumContent.vue

```diff
@@ -530,14 +531,13 @@ export default {
 	}
 
 	&__photos {
-		height: 100%;
+		height: calc(100% - 60px);
```

Comment:
This makes FileListViewer the scrollable element, but shouldn't the app content be the scrollable element?

Reply:
Ah, we probably need vue v7 for that, right?

Reply:
At this point, we don't really care which element is scrollable, no ?